src/player/Player.php

```diff
@@ -869,7 +869,7 @@ static function() : void{
 	 * Checks if the player is currently on the ground. This is more accurate than {@link Player::isOnGround()} but slower.
 	 */
 	public function isActuallyOnGround() : bool{
-		$bb = $this->boundingBox->expandedCopy(0, 0.001, 0);
+		$bb = $this->boundingBox->expandedCopy(-0.001, 0.001, -0.001);
```

Comment:
This change doesn't look related to the PR. Why would this be needed for ground calculation?

Reply:
Fixes inaccurate progress at high x/z coordinates

Reply:
Many things break down at large coordinates. I doubt this is the proper fix for the problem.